src/scenes/OrganizationProjectView.re

```diff
@@ -29,36 +29,209 @@ module Styles = {
+                ),
+              )
+            }
+          }
+          initialState={name: ""}
+          i18n=ReForm.Validation.I18n.ptBR
+          schema=[(`name, Required)]>
+          ...{({handleChange, form}) =>
+            <View style=Styles.wrapper>
+              <AppBar
+                title="Project"
+                renderLeft={_ => <BackButton onPress={_ => closeModal()} />}
+                renderRight={_ =>
+                  <DeleteButton
+                    onPress={() =>
+                      Js.Promise.(
+                        removeOrganizationProject(
+                          ~variables=
+                            RemoveOrganizationProjectMutation.make(
+                              ~input={
+                                "organizationId": organizationId,
+                                "projectId": projectId,
+                              },
+                              (),
+                            )##variables,
+                          (),
+                        )
+                        |> then_(result =>
+                             RemoveOrganizationProjectMutationContainer.convertJsInputToReason(
+                               result,
+                             ).
+                               data
+                             ->Belt.Option.map(data =>
+                                 data##removeOrganizationProject
+                               )
+                             ->Belt.Option.map(removeOrganizationProject =>
+                                 switch (removeOrganizationProject##error) {
+                                 | Some(error) =>
+                                   snackbar.send(Set(Open(error)));
+                                   Js.Promise.resolve();
+                                 | _ =>
+                                   snackbar.send(
+                                     Set(
+                                       Open(
+                                         {j|Infelizmente, nosso servidor não conseguiu processar essa requisição. Por favor, tente novamente.|j},
+                                       ),
+                                     ),
+                                   );
+                                   Js.Promise.resolve();
+                                 }
+                               )
+                             ->Belt.Option.getWithDefault(
+                                 Js.Promise.resolve(),
+                               )
+                           )
+                        |> then_(() =>
+                             snackbar.send(Set(Open("Projeto Deletado")))
+                             |> Js.Promise.resolve
+                           )
+                        |> ignore
+                      )
+                    }
+                  />
+                }
+              />
+              <View style=Styles.fieldsWrapper>
+                <Webhook text="https://summ.link/234234234234" />
```

Comment:
consume this from `Util`, `Util.webhook(id)`

Reply:
we have a mutation for it, we won't use?

Reply:
the mutation is to regenerate no?

Reply:
gotcha